addons/l10n_si/data/account_tax_report_data.xml

```diff
@@ -419,4 +420,713 @@
+        <field name="name">Slovenia VAT RC (PD-O)</field>
+        <field name="root_report_id" ref="account.generic_tax_report"/>
+        <field name="country_id" ref="base.si"/>
+        <field name="column_ids">
+            <record id="tax_report_pd_balance" model="account.report.column">
+                <field name="name">Balance</field>
+                <field name="expression_label">balance</field>
+            </record>
+        </field>
+        <field name="line_ids">
+            <record id="tax_report_pd_a3" model="account.report.line">
+                <field name="name">a3. Value of supplies of goods and services</field>
+                <field name="code">pd_a3</field>
+                <field name="expression_ids">
+                    <record id="tax_report_pd_a3_tag" model="account.report.expression">
+                        <field name="label">balance</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">pd_a3</field>
+                    </record>
+                </field>
+            </record>
+        </field>
+    </record>
+
+    <!-- IR -->
+    <record id="tax_report_ir" model="account.report">
+        <field name="name">Payable VAT (IR)</field>
+        <field name="root_report_id" ref="account.generic_tax_report"/>
+        <field name="country_id" ref="base.si"/>
+        <field name="column_ids">
+            <record id="tax_report_ir_balance" model="account.report.column">
+                <field name="name">Balance</field>
+                <field name="expression_label">balance</field>
+            </record>
+        </field>
+        <field name="line_ids">
+            <record id="tax_report_ir_7" model="account.report.line">
+                <field name="name">7. SLO Value excluding VAT</field>
+                <field name="code">ir_7</field>
+                <field name="expression_ids">
+                    <record id="tax_report_ir_7_tag" model="account.report.expression">
+                        <field name="label">balance</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">ir_7</field>
+                    </record>
+                </field>
+            </record>
+            <record id="tax_report_ir_8" model="account.report.line">
+                <field name="name">8. Self-taxation of supplies Article 76a</field>
+                <field name="code">ir_8</field>
+                <field name="expression_ids">
+                    <record id="tax_report_ir_8_tag" model="account.report.expression">
+                        <field name="label">balance</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">ir_8</field>
+                    </record>
+                </field>
+            </record>
+            <record id="tax_report_ir_oproscen_promet_eu" model="account.report.line">
```

Comment:
Use english record ids. Easier to understand :smile: (Happens several times on file)